packages/contact-center/task/src/IncomingTask/incoming-task.presentational.tsx

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {IncomingTaskPresentationalProps} from '../task.types';
+
+const IncomingTaskPresentational: React.FunctionComponent<IncomingTaskPresentationalProps> = (props) => {
+  const {currentTask, accept, decline, isBrowser, answered, ended, missed} = props;
+
+  if (!currentTask) {
+    return <></>; // hidden component
+  }
+
+  const callAssociationDetails = props.currentTask.data.interaction.callAssociatedDetails;
+  const {ani, dn, virtualTeamName} = callAssociationDetails;
+  const timeElapsed = ''; // TODO: Calculate time elapsed
```

Comment:
Will this be done in the current PR or another JIRA?

Reply:
This will be done as part of the call-control.